src/main/java/com/ably/kafka/connect/config/ConfigValueEvaluator.java

```diff
@@ -6,7 +6,26 @@
 import java.nio.charset.StandardCharsets;
 
 public class ConfigValueEvaluator {
+    /*
+    Result class wrapping the evaluated value and whether it should be skipped or not
+    This has been created to wrap the result of the evaluation of a config value when skippability is involved.
+     * */
+    public static class Result{
+        boolean skip;
+        String value;
+        public Result(boolean skip, String value) {
+            this.skip = skip;
+            this.value = value;
+        }
+
+        public String getValue() {
+            return value;
+        }
 
+        public boolean shouldSkip() {
+            return skip;
+        }
+    }
```

Comment:
I would add a blank line below this line for readability.

Reply:
110e0282f67f57de0aa6845cb9e3ddd666a1ad1a